tycho-apitools-plugin/src/main/java/org/eclipse/tycho/apitools/ApiAnalysisMojo.java

```diff
@@ -141,6 +145,30 @@ public class ApiAnalysisMojo extends AbstractMojo {
 	@Parameter(defaultValue = "false", property = "tycho.apitools.enhanceLogs")
 	private boolean enhanceLogs;
 
+	/**
+	 * Whether to create an XML report for the problems like the one created by the
+	 * <a href=
+	 * "https://help.eclipse.org/latest/topic/org.eclipse.pde.doc.user/reference/api-tooling/ant-tasks/analysis-ant-task.htm">API
+	 * Tools ANT task</a>.
+	 * <p>
+	 * Typically, this is best used in combination with
+	 * {@code <failOnError>false</failOnError>} in order to not fail the Maven build
+	 * and allow subsequent analysis by external means.
+	 *
+	 * @since 4.0.13
+	 */
+	@Parameter(defaultValue = "false", property = "tycho.apitools.writeXmlReport")
+	private boolean writeXmlReport;
+
+	/**
+	 * If {@link #writeXmlReport} is enabled, the directory where to write the
+	 * {@code report.xml} to.
+	 *
+	 * @since 4.0.13
+	 */
+	@Parameter(defaultValue = "${project.build.directory}/apianalysis")
+	private File xmlReportDirectory;
```

Comment:
```suggestion
	@Parameter(defaultValue = "${project.build.directory}/apianalysis/report.xml", property = "tycho.apitools.report")
	private File xmlReport;
```
Instead of a directory I would simply allow configuration of the whole file, also we should have a property for this.